routes/api.js

```diff
@@ -288,13 +288,14 @@ var getMembersBySkills = async function (req, res, next) {
  * @apiUse OnNotFoundError
  */
 var addMember = function (req, res, next) {
+    console.log('req body', req.body);
     try {
-        if (req.body.hasOwnProperty("memberProfile")) {
-            let newMember = req.body.memberProfile;
-            if (!newMember.hasOwnProperty("email")) {
+        if (req.body.email) {
+            let newMember = req.body;
```

Comment:
This works when the caller sends a good JSON. We need a validation of the JSON object sent. Otherwise the results will be unexpected